db/models/volunteer.js

```diff
@@ -4,7 +4,7 @@ const Sequelize = require('sequelize')
 const db = require('APP/db')
 
 const Volunteer = db.define('volunteers', {
```

Comment:
I think that is what it was. I really think we could benefit from a bit of a vocabulary clarification honestly. For me, the volunteers are like you described: people who 'just signed up' and are awaiting assignment to a team, or validation, or whatever.